Ecal/include/Ecal/EcalVetoProcessor.h

```diff
@@ -33,7 +33,8 @@
 #include <map>
 #include <memory>
 
-// ROOT (for anle calculations)
+// ROOT (for angle calculations)
+// TODO: Should be moved to the XYZ vector class
 #include "TVector3.h"
```

Comment:
Please do this now. https://root.cern.ch/doc/master/group__GenVector.html

Reply:
There is 18 occurrences of this 
https://github.com/search?q=repo%3ALDMX-Software%2Fldmx-sw%20TVector3&type=code

I'd like to have this a dedicated PR

Reply:
Made an issue: https://github.com/LDMX-Software/ldmx-sw/issues/1780